### PR TITLE
Added IT Passport Test Cases

### LINKED ISSUE #21: Detect IT_PASSPORT
### Description
AN IT_PASSPORT (Italian Passport) includes a distinct nine-character string assigned by the the Italian Citizenship Assistance (ICA). There are two types of IT_PASSPORTs: ordinary and temporary. It is important to detect IT_PASSPORT in order to prevent personal information from being leaked and to prevent identity fraud and theft. For more information, click [here](https://italiancitizenshipassistance.com/the-italian-passport/).
### Detection Steps
An IT_PASSPORT consists of two digits or letters (not case-sensitive) followed by seven digits with no spaces or delimiters.

Example: JF3349917

### PR DESCRIPTION
**Pull request by everett-alanHood**

### CODE REVIEW

**Review by claesmk — APPROVED** [review]
The goal for this assignment was to test the analyze_text() function, not to create a custom regular expression.  I've updated your code to demonstrate how to do this.

File: test_team_tech_bros.py [hunk 1]
```diff
@@ -21,6 +21,27 @@ def test_it_identity_card(self):
...
+        
+        # Test valid passport with context
+        result = analyze_text(f"My italiano passaporto number is {p_num}", ['IT_PASSPORT'])
+        print(result)
+        self.assertEqual('IT_PASSPORT', result[0].entity_type)
+        self.assertAlmostEqual(0.4, result[0].score, places=2)
+
+        # Test valid passport without context
+        result = analyze_text(f"My number is {p_num}", ['IT_PASSPORT'])
+        print(result)
```
Comment (NCATAGGIES) [inline test_team_tech_bros.py:36]: Remove Print Statements: The print(result) statements are not needed in unit tests. They can clutter test output and are typically not used in automated testing environments. If you need to debug, consider using logging or assert statements to verify expected conditions instead.

File: test_team_tech_bros.py [hunk 2]
```diff
@@ -21,6 +21,27 @@ def test_it_identity_card(self):
     def test_it_passport(self):
         """Test IT_PASSPORT functionality"""
 
+        # Valid examples
+        p_num = "JF3349917"
+        invalid_p = "12ABC5678"
+        
+        # Test valid passport with context
+        result = analyze_text(f"My italiano passaporto number is {p_num}", ['IT_PASSPORT'])
+        print(result)
```
Comment (NCATAGGIES) [inline test_team_tech_bros.py:30]: Remove Print Statements: The print(result) statements are not needed in unit tests. They can clutter test output and are typically not used in automated testing environments. If you need to debug, consider using logging or assert statements to verify expected conditions instead.

File: test_team_tech_bros.py
Comment (NCATAGGIES) [inline test_team_tech_bros.py:1]: Variety of Test Cases: The tests include valid examples with context, valid examples without context, and invalid examples, which is essential for thorough testing.
Appropriate Assertions: The use of assertEqual, assertAlmostEqual, and assertListEqual is appropriate for validating expected outcomes.

**Review by NCATAGGIES — COMMENTED** [review]
Your code for testing the IT_PASSPORT functionality is well-structured and covers key scenarios.

**Review by NCATAGGIES — APPROVED** [review]

**Review by mawdogbe — COMMENTED** [review]
Looks good, nice changes Everett.

**Pull request merged**